robotbase/src/main/java/frc/robot/commands/drive/DriveToApriltag.java

```diff
@@ -79,10 +90,17 @@ public void execute() {
     if (Utility.isWithinTolerance(ty, m_tyOffset, Constants.SWERVE.APRILTAG_Y_TOLERANCE)) {
       ty = m_tyOffset;
     }
-    Robot.swerve.drive(
-        -m_yController.calculate(ty + Constants.SWERVE.APRILTAG_TY_MAGIC_OFFSET),
-        -m_xController.calculate(tx),
-        m_rotationController.calculate(rotationError));
+    if (m_invertSpeeds) {
+      Robot.swerve.drive(
+          m_yController.calculate(ty + Constants.SWERVE.APRILTAG_TY_MAGIC_OFFSET),
```

Comment:
Doing a lot of repetitive stuff in these if statements. Just calculate and invert the output once.

xoutput = m_xcontroller.calculate(tx) * m_invertSpeeds ? 1 : -1